lib/log.rb

```diff
@@ -163,6 +171,12 @@ def print(msg)
       @@puts_method.call(msg)
     end
 
+    def filter_custom_logs(msg)
+      # This is to capture the message when the trickier failure like below seen in any test case execution.
+      if (msg =~/connection to the server\S+was refused|Unable to connect to the server|Error from server \(ServiceUnavailable\)/)
```

Comment:
If we are `Unable to connect to the server`, will the debug step `oc get node` works ?